packages/trip-form/src/TripOptions/OptionButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import * as S from "./styled";
+// eslint-disable-next-line prettier/prettier
```

Comment:
What is this for?

Reply:
Prettier wasn't handling typescript imports correctly. I think it's due to the failing `core-utils` import. Comment added in 775500dafb258ffc2fc5d431b8619ebe3f21b54f